src/app/auth/reset-password/reset-password.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnInit } from '@angular/core';
+import { AuthValidationService } from 'src/app/core/auth-validation.service';
+import { AuthService } from 'src/app/core/auth.service';
+import { ActivatedRoute } from '@angular/router';
+import { FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'cc-reset-password',
+  templateUrl: './reset-password.component.html',
+  styleUrls: ['./reset-password.component.scss']
+})
+export class ResetPasswordComponent implements OnInit {
+  errorMessage!: String;
+  showError!: Boolean;
```

Comment:
```suggestion
  showError!: boolean;
```
Boolean is an Object type